src/domain/application/hooks/useApplicationSendNotification.tsx

```diff
@@ -14,8 +14,12 @@ export default function useApplicationSendNotification() {
       dismissible: true,
       autoClose: true,
       autoCloseDuration: 8000,
-      label: t('hakemus:notifications:sendSuccessLabel'),
-      message: t('hakemus:notifications:sendSuccessText'),
+      label: isMuutosilmoitus
```

Comment:
These label and message are not displayed correctly for muutosilmoitus:

<img width="360" alt="Screenshot 2025-03-24 at 14 32 47" src="https://github.com/user-attachments/assets/2363e8af-9db0-44d7-8aab-0b5b975cd264" />

Reply:
Turned out this was a problem on my end. When I restarted the app this was fixed.